### PR TITLE
feat: add files relevant for the community

### PR DESCRIPTION
**Pull request by jvanz**
## Description

As the central point for documentation for the community, this commit moves relevant files into the repository.

Fix https://github.com/kubewarden/kubewarden-controller/issues/685

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# CONTRIBUTING
+
+Thanks for your interest in contributing to Kubewarden.
+
+There are different ways you could help us simplify the adoption of policy as code.
+
+## Feedback
+
+* Starring our main repository [kubewarden/policy-server](https://github.com/kubewarden/policy-server). GitHub stars do matter !
```
Comment (viccuad) [inline CONTRIBUTING.md:9]: wasn't it kubewarden-controller?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## Code 
+
+Contributing to our [policy templates](https://github.com/topics/kubewarden-policy-template), [policy SDKs](https://github.com/topics/kubewarden-policy-sdk) and [policies](https://github.com/topics/kubewarden-policy).
+You could also look into the following "core" projects:
+
+| Project | Scope | Language |
+|---------|---------|--------|
+| [`kubewarden-controller`](https://github.com/kubewarden/kubewarden-controller) | Kubernetes integration point| Go |
+| [`policy-server`](https://github.com/kubewarden/policy-server) | Run Kubewarden policies | Rust |
+| [`kwctl`](https://github.com/kubewarden/kwctl) | Kubewarden policy multi-purpose cli tool | Rust |
```
Comment (viccuad) [inline CONTRIBUTING.md:22]: I wonder if we can add a pointer here to a page that has the core projects for DRY.

If not, 


```suggestion
| [`kwctl`](https://github.com/kubewarden/kwctl) | Kubewarden policy multi-purpose cli tool | Rust |
[`audit-scanner`](https://github.com/kubewarden/audit-scanner) | continously report policy results | Go |
```

**Review by viccuad — COMMENTED** [review]
We should DRY; expanded acceptance criteria of https://github.com/kubewarden/kubewarden-controller/issues/685 to remove old files and links when this is merged.

Have some quibbles that I would like in before approving.

**Comment by viccuad** [comment]
can we add https://github.com/kubewarden/rfc/blob/main/rfc/0013-governance.md to the PR?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jvanz) [reply CONTRIBUTING.md:9]: Yes. Fun fact, this is wrong in our current CONTRIBUTING

**Comment by jvanz** [comment]
> can we add https://github.com/kubewarden/rfc/blob/main/rfc/0013-governance.md to the PR?

Oh, I was looking for the governance file. I did not remember that is a RFC. Nice! I'll added.

**Review by flavio — APPROVED** [review]

**Review by viccuad — APPROVED** [review]

**Pull request merged**